docs/full-mesh.md

```diff
@@ -77,17 +77,19 @@ spec:
       enabled: true
   deployments:
   - name: activator
-    annotations:
+    labels:
       "sidecar.istio.io/inject": "true"
+    annotations:
       "sidecar.istio.io/rewriteAppHTTPProbers": "true"
   - name: autoscaler
-    annotations:
+    labels:
       "sidecar.istio.io/inject": "true"
+    annotations:
       "sidecar.istio.io/rewriteAppHTTPProbers": "true"
 EOF
 ```
 
-Then, create KnativeService with `sidecar.istio.io/inject: "true"`, `sidecar.istio.io/rewriteAppHTTPProbers: "true"` and `serving.knative.openshift.io/enablePassthrough: "true"` annotations in your namespace,
+Then, create KnativeService with the `sidecar.istio.io/inject: "true"` label and `sidecar.istio.io/rewriteAppHTTPProbers: "true"`, `serving.knative.openshift.io/enablePassthrough: "true"` annotations in your namespace,
 which is one of the namespaces in the `ServiceMeshMemberRoll`.
```

Comment:
These doc files are not source of truth for any public content right?

Reply:
Yes, not sure why they are there in the first place. Just updated every reference I could find.